### ISSUE TITLE
AppCenter fails to initialise in .NET6 macOS app

### ISSUE THREAD

**Issue by tipa** [microsoft/appcenter-sdk-dotnet#1690]
### **Description**

As [requested here](https://github.com/microsoft/appcenter-sdk-dotnet/issues/1647#issuecomment-1274230505), I am filing this new issue. The AppCenter SDK throws an exception when trying to initialise it in a .NET6 macOS app (it's NOT a MAUI app).

```
Could not create an native instance of the type 'Microsoft.AppCenter.Apple.Bindings.MSACWrapperSdk': the native class hasn't been loaded.
It is possible to ignore this condition by setting ObjCRuntime.Class.ThrowOnInitFailure to false.

   at Foundation.NSObject.InitializeObject(Boolean alloced)
   at Microsoft.AppCenter.Apple.Bindings.MSACWrapperSdk..ctor(String wrapperSdkVersion, String wrapperSdkName, String wrapperRuntimeVersion, String liveUpdateReleaseLabel, String liveUpdateDeploymentKey, String liveUpdatePackageHash)
   at Microsoft.AppCenter.AppCenter.SetWrapperSdk()
   at Microsoft.AppCenter.AppCenter.PlatformStart(String appSecret, Type[] services)
   at AppCenterTest.AppDelegate.DidFinishLaunching(NSNotification notification) in /Users/timopartl/Desktop/AppCenterTest/AppDelegate.cs:line 15
```
### **Repro Steps**

1. Create a .NET6 app with macOS target
2. Add the `Microsoft.AppCenter.Analytics` & `Microsoft.AppCenter.Crashes` dependency (5.0.0-preview.1)
3. Call `AppCenter.Start("{{secret}}", typeof(Analytics), typeof(Crashes));` in `DidFinishLaunching`

### **Details**

1. What is your app platform (Xamarin.Android or Xamarin.iOS or UWP)?
    - .NET6 macOS
2. Which SDK version are you using?
    - 5.0.0-preview.1
4. Which OS version did you experience the issue on?
    - macOS 12.6
5. What device version did you see this error on?  Were you using an emulator or a physical device?
    - Mac mini with M1 chip

### **Logs**
I enabled verbose logging using `AppCenter.LogLevel = LogLevel.Verbose` but no data showed in the output window

### **Example project**
[AppCenterTest.zip](https://github.com/microsoft/appcenter-sdk-dotnet/files/9753309/AppCenterTest.zip)

**Comment by DmitriyKirakosyan** [comment]
Thank you for this issue and the test project! I was able to repro it. We'll keep you informed about any updates.

**Comment by MGohil** [comment]
This was also giving me an exception in my iOS application on startup while debugging on iOS simulator.

![image](https://user-images.githubusercontent.com/16605284/195783759-48a8b255-63a6-4f7d-a9e0-91df9586262c.png)

However, when I set "Use the LLVM optimizing compiler" to true, this worked for me in iOS and I could see the Analytics in AppCenter for my iOS app. 

![image](https://user-images.githubusercontent.com/16605284/195784040-2eb08531-0939-4d91-a507-6a7ff0fb4ab1.png)

**Comment by 7702244** [comment]
I have the same problem on my MacBook Pro M1 Max.

**Comment by DmitriyKirakosyan** [comment]
Fixed in 5.0.0.
